sim/side.ts

```diff
@@ -629,19 +629,25 @@ export class Side {
 
 			switch (choiceType) {
 			case 'move':
-				let targetLoc = 0;
+				const original = data;
+				const error = () => this.emitChoiceError(`Conflicting arguments for "move": ${original}`);
+				let targetLoc: number | undefined;
 				let megaOrZ = '';
 				while (true) {
 					if (/\s-?[1-3]$/.test(data)) {
+						if (targetLoc !== undefined) return error();
 						targetLoc = parseInt(data.slice(-2), 10);
```

Comment:
So can you add a check for Conversion 2?

Reply:
Yeah, I'm doing that now. :) I assumed you'd notice it wasn't present and wouldn't merge without it, and I wanted to give you a chance to respond.

Reply:
Done.